snuba_sdk/visitors.py

```diff
@@ -157,8 +151,12 @@ def _stringify_scalar(self, value: ScalarType) -> str:
             else:
                 decoded = value
 
-            decoded = unescaped_quotes.sub("\\'", decoded)
-            decoded = unescaped_newline.sub("\\\\n", decoded)
+            # The ' and \ character are escaped in the string to ensure
+            # the query is valid. They are de-escaped in the SnQL parser.
+            # Also escape newlines since they break the SnQL grammar.
+            decoded = (
```

Comment:
You can replace multiple characters with a single pass with `maketrans` 
https://www.w3schools.com/python/ref_string_translate.asp